src/ExtendedView.js

```diff
@@ -1,30 +1,50 @@
+import React from 'react'
+import {fetchSingleMoviePoster} from './apiCalls'
 import './ExtendedView.css'
 
-function ExtendedView(props) {
+class ExtendedView extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      singleMovieData: {},
+      error: ''
+    }
+  }
 
+  componentDidMount() {
+    fetchSingleMoviePoster(this.props.id)
+      .then(data => {
+        this.setState({
+           singleMovieData: data.movie
+        })
+      })
+      .catch(error => this.setState({error: error}))
+  }
 
-  return(
-    <div className="extended-page">
-      <button className="back" onClick={() => props.changeExtendedState()}>BACK</button>
-        <div className="movie-poster">
-          <div
-            className="movie-image"
-            style={{backgroundImage: `url(${props.singleMovie.backdrop_path})`, backgroundSize: "cover", height: "75vh"}}>
-            <p className="title">{props.singleMovie.title}</p>
-            <button className="rating">Rating: {((props.singleMovie.average_rating/10)*100).toFixed(0)}%</button>
-          </div>
-        </div>
-          <section className="movie-info">
-            <div className="date-time">
-              <p className="release-date">{(props.singleMovie.release_date)}</p>
-              <p className="genre">{props.singleMovie.genres}</p>
-              <p className="runtime">{props.singleMovie.runtime} minutes</p>
+  render() {
+    return(
+      <div className="extended-page">
+          <div className="movie-poster">
+            <div
+              className="movie-image"
+              style={{backgroundImage: `url(${this.state.singleMovieData.backdrop_path})`, backgroundSize: "cover", height: "75vh"}}
+            >
+              <p className="title">{this.state.singleMovieData.title}</p>
+              <button className="rating">Rating: {((this.state.singleMovieData.average_rating/10)*100).toFixed(0)}%</button>
```

Comment:
Think about cleaning this data before setting to state? Also for release date - if we want to change this to just year.